erts/doc/src/erl.xml

```diff
@@ -627,6 +627,33 @@
         <p>Sets the default binary virtual heap size of processes to the size
           <c><![CDATA[Size]]></c>.</p>
       </item>
+      <marker id="+hmax"/>
+      <tag><c><![CDATA[+hmax Size]]></c></tag>
+      <item>
+        <p>Sets the default maximum heap size of processes to the size
```

Comment:
You probably want to have something here saying that +hmax defaults to being disabled (being set to 0).  That isn't clear from the other documentation.